src/Symfony/Component/DependencyInjection/Tests/Fixtures/Container/NoConstructorContainer.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Symfony\Component\DependencyInjection\Tests\Fixtures\Container;
 
-class NoConstructorContainer
+use Symfony\Component\DependencyInjection\Container;
+
+class NoConstructorContainer extends Container
```

Comment:
Should we create a new test case to still keep the existing one (a class without a constructor not inheriting one from its base class)?

Reply:
I was also thinking about that .. and to me the old test just seemed incomplete since that class didn't even implement the interface.